src/components/Sidebar/Sidebar.tsx

```diff
@@ -22,10 +22,19 @@ interface Props {
 }
 
 const Sidebar: React.FC<Props> = ({ isComparisonMode, title }) => {
+
+  
   const { comparison } = useComparison();
   const { selected } = useSelectedDistrict();
   const { isSidebarOpen, setIsSidebarOpen } = useSidebar();
 
+  useEffect(() => {
+    console.log("aaaaaaaaaaaaaaaaaaaaaaaaa")
```

Comment:
Não estou responsável pela revisão, mas achei estranho esse console.log e o console.log da linha 36. Acredito que o autor do PR tenha esquecido esses dois console.logs. Apenas sinalizando para caso o/a dev tenha esquecido.

Reply:
Please, be free to review any time you want